composer.json

```diff
@@ -12,6 +12,7 @@
         "zendframework/zendsearch": "2.*@dev",
         "symfony/monolog-bundle": "^2.8.7 || ^3.0",
         "doctrine/doctrine-bundle": "^1.4",
+        "doctrine/data-fixtures": "^1.1",
```

Comment:
why is this needed?

Reply:
Because the `Doctrine\Common\DataFixtures\Purger\ORMPurger` is necessary in the tests and the library was removed from a dependency (I'm not sure which one it was)